spec/rubocop/cop/rspec/nested_groups_spec.rb

```diff
@@ -35,11 +54,32 @@
     expect(cop.config_to_allow_offenses[:exclude_limit]).to eq('Max' => 4)
   end
 
-  it 'ignores non-spec context methods' do
```

Comment:
It wasn't ignoring. Just the example didn't have enough nesting to trigger the offence.